tests/kernel/fpu_sharing/float_disable/src/k_float_disable.c

```diff
@@ -263,9 +263,11 @@ void test_k_float_disable_irq(void)
 	zassert_true(ok, "");
 }
 #else
-void test_k_float_disable_irq(void)
+ZTEST(k_float_disable, test_k_float_disable_irq)
```

Comment:
Personal suggestion: Put the #if / #else to info function definition of k_float_disable,  you don't have to write ZTEST(k_float_disable,....)  twice.

Reply:
For this #if /#else, we have some #include and other functions inside, those are not suitable to put into function test_k_float_disable_irq

Reply:
> Personal suggestion: Put the #if / #else to info function definition of k_float_disable, you don't have to write ZTEST(k_float_disable,....) twice.

For this #if /#else, we have some #include and other functions inside, those are not suitable to put into function test_k_float_disable_irq

Reply:
Hi @yperess , may I ask you a question? Any suggestion of  "should we keep one ZTEST function definition as possible or it's ok for both"? Thanks!